### PR TITLE
Initial draft

### PR DESCRIPTION
**Pull request by Kralizek**
Sketching down the basic structure of the tool.

- Added .devcontainer configuration
- Added blank solution file
- Configuration for Visual Studio Code
- First draft of the program
- Added helper programs

### CODE REVIEW

**Review by simgu — APPROVED** [review]
Looks good! Ship it! :)

File: README.md [hunk 1]
```diff
@@ -33,7 +33,23 @@ The following options will be supported
 
 |Option|Short version|Required|Description|
 |-|-|-|-|
-|`--bucket`||true|The name of the bucket to place the lock file|
-|`--prefix`||true|The prefix to be prepended to the key of the lock file|
+|`--bucket`|||The name of the bucket to place the lock file|
+|`--prefix`|||The prefix to be prepended to the key of the lock file|
 |`--token`|||The token to be used to ensure uniqueness.<br />If not specified, an hash of the executable is used|
```
Comment (elindanielsson) [inline README.md:38]: Do we need to specify the region as well?

**Review by elindanielsson — COMMENTED** [review]
Looks good, I just had one question about the region for the bucket to use.
Do we need the possibility to specify it here, if you want to use a different region for the bucket than the region used for the app?

File: README.md (hunk 1 above)
Comment (Kralizek) [reply README.md:38]: Created issue to discuss this: #3

**Pull request merged**